layouts/_default/baseof.html

```diff
@@ -77,7 +77,7 @@
     {{ hugo.Generator }} {{ $style := resources.Get "sass/main.scss" | css.Sass | resources.Minify }}
     <link rel="stylesheet" href="{{ $style.Permalink }}" />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap"
```

Comment:
I don't think this makes a big difference but these weights are unused so might as well remove

Reply:
👍